fixtures/flight/server/global.js

```diff
@@ -121,9 +127,9 @@ app.all('/', async function (req, res, next) {
         buildPath = path.join(__dirname, '../build/');
       }
       // Read the module map from the virtual file system.
-      const moduleMap = JSON.parse(
+      const ssrBundleConfig = JSON.parse(
         await virtualFs.readFile(
-          path.join(buildPath, 'react-ssr-manifest.json'),
+          path.join(buildPath, 'react-ssr-bundle-config.json'),
```

Comment:
The name manifest comes from the Webpack usage. That just leaked into our internal SSRManifest name. Renaming it is a bit the tail wagging the dog. We shouldn't try to change the naming convention for these files which are still just manifests.

Reply:
In other words, our SSRManifest was a Module Map living in a Manifest file. Now it's a Bundler Config living in a Manifest file. It doesn't change that the outer thing is still a manifest.

Reply:
I updated the filename but also renamed our type `SSRManifest` to `SSRModuleMap` since that is what it is. We can go back to the manifest type if we consolidate ModuleLoading with the ModuleMap as you questioned below however since you previously suggested doing the opposite I'll await further consideration there.